### DISCUSSION TITLE
New extension: Jellyfin/Emby Notify

### DISCUSSION THREAD

**Discussion by luckedea** [nzbgetcom/nzbget#297] [Ideas]
A request from reddit https://www.reddit.com/r/usenet/comments/1cn6rq8/comment/l6x3uo2/
It doesn't look a similar extension exists, and that's something we can add once a few users would show activity on this discussion.

**Comment by Sparklingx** [comment]
I think this is a necessity.  And not everyone uses plex.

**Reply by luckedea** [reply]
> I think this is a necessity. And not everyone uses plex.
Personally I actually use both.

**Comment by dnzbk** [comment]
The new NotifyEmbyJellyfin extension is [released](https://github.com/nzbgetcom/Extension-NotifyEmbyJellyfin) - feel free to test it.
Can be installed by the extension manager.

**Reply by Brfalo** [reply]
Much Thanks for this one!  Though I have plex as a backup, I use emby way more (so I get why folks might have both).  I can verify that on my setup this Emby\Jellyfin script works great and Emby immediately pulls in the new files.  I did have some firewall issues with the main.py (which crashed Nzbget once) but once I ok'd that process action in the firewall,  this script was able to work without issue in Post-Processing.  I also did have to add a dedicated Emby api key for just nzbget, and of course mirrored that new key in the scripts setup field.   

Using Nzbget version 24.3 | Emby premiere v4.8.10.0 |  Win 7 Pro
